### PR TITLE
added the ability to make commands only for the server owner

### PR DESCRIPTION
**Pull request by OneTrueBen**

### CODE REVIEW

File: modrole.py [hunk 1]
```diff
@@ -29,25 +29,31 @@ async def wrapped(*args):
...
 
+def server_owner_only():
+    def wrapper(func):
+        @functools.wraps(func)
+        async def wrapped(*args):
+            ctx = next(a for a in args if type(a) is commands.context.Context)
+            if ctx.author.id == ctx.guild.owner_id:
+                return await func(*args)
+            else:
+                await ctx.send('no!!!!')
```
Comment (benjjs) [inline modrole.py:44]: I can think of more informative things to say in this case.

**Review by benjjs — APPROVED** [review]
Sure

File: modrole.py (hunk 1 above)
Comment (OneTrueBen) [reply modrole.py:44]: cool

Comment (OneTrueBen) [reply modrole.py:44]: it's a stylistic decision

**Pull request merged**